distribution/client/test/healthchecker.test.ts

```diff
@@ -64,52 +64,66 @@ describe('The healthchecker module', () => {
     it('should pass cases', async (done) => {
       // ugly hack to wait a bit for the server to start...
       setTimeout( async () => {
-        const healthChecker = new HealthChecker(`http://localhost:${port}`, {delay: 100, retry: 1});
         serverOptions.instanceIdentity = true;
         serverOptions.metrics.plugins = true;
         serverOptions.metrics.deadlock = true;
         serverOptions.metrics.body = true;
-
-        let result = await healthChecker.check();
-        expect(result.healthy).toBe(true);
-        expect(result.message).not.toContain('wrong');
+        const healthChecker = new HealthChecker(`http://localhost:${port}`, {delay: 300, retry: 1});
+        expect(await healthChecker.check()).toBeTruthy();
 
         // broken /instance-identity page
         serverOptions.instanceIdentity = false;
-        result = await healthChecker.check();
-        expect(result.healthy).toBe(false);
+        try {
```

Comment:
I believe it is better to rely on Jest for this behavior, e.g.:

```typescript
expect(async () => {
  await healthChecker.check();
}).toThrow(SomeError);
```

If this doesn't quite work properly due to the async nature of  the call, I believe adding `return` would suffice, e.g.

```typescript
return expect(async () => { }).toThrow()
```

That way Jest gets a Promise to resolve for the test.

Reply:
I tried many forms, to no easy/obvious avail for now. 

How about I address this in a followup? I would also like to take the opportunity to split that crappy (that I wrote) test in many smaller ones as much as possible.

Reply:
Fine by me to address later.